test/topology/conftest.py

```diff
@@ -63,7 +63,7 @@ def on_error(exception, *_):
 
 
 def run_async(self, *args, **kwargs) -> asyncio.Future:
-    return _wrap_future(self.execute_async(*args, **kwargs))
+    return _wrap_future(self.execute_async(*args, **kwargs, timeout=60.0))
```

Comment:
I would do that with a bit more care, you are new blocking and call from passing it's own timeout

Check the kwargs first and then if it doesn't have timeout, put the default
```python
kwargs.setdefault("timeout", 60.0)
```

Reply:
Done in latest